src/material/slider/slider-thumb.html

```diff
@@ -1,10 +1,7 @@
-<div class="mdc-slider__value-indicator-container" *ngIf="discrete" #valueIndicatorContainer>
+<div class="mdc-slider__value-indicator-container" [ngClass]="{'mdc-slider__thumb--with-indicator': _isValueIndicatorVisible}" *ngIf="discrete" #valueIndicatorContainer>
   <div class="mdc-slider__value-indicator">
     <span class="mdc-slider__value-indicator-text">{{valueIndicatorText}}</span>
   </div>
 </div>
 <div class="mdc-slider__thumb-knob" #knob></div>
-<div
-  matRipple
-  class="mat-mdc-focus-indicator"
-  [matRippleDisabled]="true"></div>
+<div matRipple class="mat-mdc-focus-indicator" [matRippleDisabled]="true"></div>
```

Comment:
It is disabled because we only want it to be triggered programmatically. This is because the slider ripples behave differently from regular ripples according to the material design spec

Reply:
I think that doing it this way could be problematic, because the ripples for focus/hover indication won't show up if ripples are disabled globally.